server/sonar-server/src/main/java/org/sonar/server/rule/ws/RulesWsParameters.java

```diff
@@ -0,0 +1,102 @@
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.server.rule.ws;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.Set;
+
+public class RulesWsParameters {
+  public static final String PARAM_REPOSITORIES = "repositories";
+  public static final String PARAM_RULE_KEY = "rule_key";
+  public static final String PARAM_ACTIVATION = "activation";
+  public static final String PARAM_QPROFILE = "qprofile";
+  public static final String PARAM_SEVERITIES = "severities";
+  public static final String PARAM_AVAILABLE_SINCE = "available_since";
+  public static final String PARAM_STATUSES = "statuses";
+  public static final String PARAM_LANGUAGES = "languages";
+  public static final String PARAM_TAGS = "tags";
+  public static final String PARAM_TYPES = "types";
+  public static final String PARAM_INHERITANCE = "inheritance";
+  public static final String PARAM_ACTIVE_SEVERITIES = "active_severities";
+  public static final String PARAM_IS_TEMPLATE = "is_template";
+  public static final String PARAM_TEMPLATE_KEY = "template_key";
+  public static final String PARAM_ORGANIZATION = "organization";
+  public static final String PARAM_COMPARE_TO_PROFILE = "compareToProfile";
+
+  public static final String FIELD_REPO = "repo";
+  public static final String FIELD_NAME = "name";
+  public static final String FIELD_CREATED_AT = "createdAt";
+  public static final String FIELD_SEVERITY = "severity";
+  public static final String FIELD_STATUS = "status";
+  public static final String FIELD_INTERNAL_KEY = "internalKey";
+  public static final String FIELD_IS_TEMPLATE = "isTemplate";
+  public static final String FIELD_TEMPLATE_KEY = "templateKey";
+  public static final String FIELD_TAGS = "tags";
+  public static final String FIELD_SYSTEM_TAGS = "sysTags";
+  public static final String FIELD_LANGUAGE = "lang";
+  public static final String FIELD_LANGUAGE_NAME = "langName";
+  public static final String FIELD_HTML_DESCRIPTION = "htmlDesc";
+  public static final String FIELD_MARKDOWN_DESCRIPTION = "mdDesc";
+  public static final String FIELD_NOTE_LOGIN = "noteLogin";
+  public static final String FIELD_MARKDOWN_NOTE = "mdNote";
+  public static final String FIELD_HTML_NOTE = "htmlNote";
+
+  /**
+   * @deprecated since 5.5, replaced by {@link #FIELD_DEFAULT_REM_FUNCTION}
+   */
+  @Deprecated
+  public static final String FIELD_DEFAULT_DEBT_REM_FUNCTION = "defaultDebtRemFn";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)